src/lib.rs

```diff
@@ -125,6 +164,15 @@ pub fn collect_stars(config: Config) -> Result<(), Box<dyn error::Error>> {
     }
     println!("Collected {} stars", stars.len());
 
+    if print_auth_warning {
+        eprintln!("\nRequest completed without authentication, re-run and provide token using `--token <auth-token>` to increase your requests from 60 to 5000 requests per hour.");
+    }
+
+    match remaining {
+        10 | 0...5 => eprintln!("Warning: You have {} out of {} requests remaining. Your request limit will reset in {} minutes.", remaining, total, reset_time.duration_since(SystemTime::now())?.as_secs()/60),
+        _ => println!("Warning: You have {} out of {} requests remaining. Your request limit will reset in {} minutes.", remaining, total, reset_time.duration_since(SystemTime::now())?.as_secs()/60),  
```

Comment:
Is this intentional? In a previous commit you had

```rust
_ => (), 
```

but how it is now will print the warning message (to `stdout`) every time, regardless of the value of `remaining`.

Reply:
Woops! Put that in for testing and forgot to remove.